app/models/delay_henka/scheduled_change.rb

```diff
@@ -11,8 +11,9 @@ class ScheduledChange < ApplicationRecord
     belongs_to :changeable, polymorphic: true
 
     validates :submitted_by_id, :attribute_name, presence: true
+    validates :schedule_at, presence: true, on: :update
```

Comment:
You mean create?

Reply:
> You mean create?

if no `on: :update`, it would fail, when create record which's schedule_at is nil, because the validates run before the set_default_schedule_at.

But yes, we can remove the callback and change the validates also.